docs/pages/config.mdx

```diff
@@ -5,19 +5,66 @@ This file needs to be named `mud.config.ts` and be in the same folder as your `f
 
 The config is used to define:
 
-- The tables in your project in the `tables` object of your configuration.
-- The [namespace](/world/namespaces-access-control) that [systems](/world/systems) and tables will be deployed in.
+- The tables in your project.
+- The [namespace(s)](/world/namespaces-access-control) that [`System`s](/world/systems) and tables will be deployed in.
 - The `System`s in your project.
   By default, the deployer will find all Solidity matching `*System.sol` (so any file ending in `System.sol`, in any folder) and deploy them as public `System`.
   If you want greater control over your systems (to change their public access or their name), you can use the `systems` object in the config.
 - The [modules](/world/modules) that will be installed in the `World`.
+- The enumerations in your `World`.
+- Deployment and code generation options.
 
 ## Multiple namespaces
 
 Starting with version 2.1 you can put multiple namespaces in the same config file, you create a `namespaces` record and within it a record for every namespace.
 For example:
 
-```tsx
+### Sample `mud.config.ts` files
+
+#### A namespace
+
+This is an extremely simple configuration file with a single namespace that contains a single table.
+The `System`s are defined implicitly, as all the contracts that match `src/namespaces/app/systems/*System.sol`.
```

Comment:
it's not required for systems to be under `systems` dir, only that they be named `*System.sol`